.planning/phases/05-test-coverage/05-02-PLAN.md

```
@@ -0,0 +1,207 @@
+      exports: ["Handle_WhenUserExists_AndEmailFree_UpdatesEmailAndReturnsSuccess",
+                "Handle_WhenUserNotFound_ThrowsNotFoundException",
+                "Handle_WhenEmailTakenByAnotherUser_ThrowsDomainException",
+                "Handle_WhenEmailBelongsToSameUser_UpdatesSuccessfully"]
+  key_links:
+    - from: "UpdateEmailRequestHandlerTest.cs"
+      to: "CoffeePeek.Account.Application/Features/User/UpdateUserProfile/UpdateEmail/UpdateProfileEmailHandler.cs"
+      via: "UpdateEmailRequestHandler.Handle(...) прямой вызов"
+      pattern: "UpdateEmailRequestHandler\\.Handle"
+---
+
+<objective>
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)